autonomi/src/client/data_types/chunk.rs

```diff
@@ -208,7 +204,7 @@ impl Client {
         &self,
         chunk: &Chunk,
         payment: ProofOfPayment,
-    ) -> Result<(), PutError> {
+    ) -> Result<ChunkAddress, PutError> {
```

Comment:
The returned address is the same as the address passed with `&Chunk`. I was thinking we might eventually return more information about the upload (e.g. costs), but we're already allowing devs to track that with the client events (though sparsely documented)?

Reply:
Yeah maybe, although in this case the payment is provided as a parameter. 
Im trying to make our api more uniform and have put return:
- price
- address where to get it